board/freescale/mx6sxsabresd/mx6sxsabresd.c

```diff
@@ -265,9 +291,76 @@ static int setup_lcd(void)
 	/* Set Brightness to high */
 	gpio_request(IMX_GPIO_NR(6, 4), "lcd_bright");
 	gpio_direction_output(IMX_GPIO_NR(6, 4) , 1);
+}
 
-	return 0;
+static void enable_lvds(struct display_info_t const *dev)
+{
+	int ret;
+
+	ret = enable_lcdif_clock(dev->bus, 1);
+	if (ret) {
+		printf("Enable LCDIF clock failed, %d\n", ret);
+		return;
+	}
+	ret = enable_lvds_bridge(dev->bus);
+	if (ret) {
+		printf("Enable LVDS bridge failed, %d\n", ret);
+		return;
+	}
+
+	imx_iomux_v3_setup_multiple_pads(lvds_ctrl_pads,
+							ARRAY_SIZE(lvds_ctrl_pads));
+
+	/* Enable CABC */
+	gpio_request(IMX_GPIO_NR(4, 18), "CABC enable");
+	gpio_direction_output(IMX_GPIO_NR(4, 18) , 1);
+
+	/* Set Brightness to high */
+	gpio_request(IMX_GPIO_NR(6, 3), "lvds backlight");
+	gpio_direction_output(IMX_GPIO_NR(6, 3) , 1);
 }
+
+struct display_info_t const displays[] = {{
+	.bus	= LCDIF2_BASE_ADDR,
+	.addr	= 0,
+	.pixfmt	= 18,
+	.detect	= NULL,
+	.enable	= enable_lvds,
+	.mode	= {
+		.name           = "Hannstar-XGA",
+		.refresh        = 60,
+		.xres           = 1024,
+		.yres           = 768,
+		.pixclock       = 15384,
+		.left_margin    = 160,
+		.right_margin   = 24,
+		.upper_margin   = 29,
+		.lower_margin   = 3,
+		.hsync_len      = 136,
+		.vsync_len      = 6,
+		.sync           = FB_SYNC_EXT,
+		.vmode          = FB_VMODE_NONINTERLACED
+} }, {
+	.bus = MX6SX_LCDIF1_BASE_ADDR,
+	.addr = 0,
+	.pixfmt = 24,
+	.detect = NULL,
+	.enable	= setup_lcd,
+	.mode	= {
+		.name			= "MCIMX28LCD",
```

Comment:
Alignment with the rest of the list.
```suggestion
		.name           = "MCIMX28LCD",
```